test/aggregation-batching.ts

```diff
@@ -54,6 +55,15 @@ describe('when testing a batched aggregation service', function () {
   hookServersStartStop({ skipEarthdataLogin: false });
   const collection = 'C1243729749-EEDTEST';
   describe('with only one batch that should be created', function () {
+    let batchInputsStub;
```

Comment:
It doesn't look like this part is needed (the test was passing before).

Reply:
I added this just to be explicit about what values were getting set.